src/h5web/LoadingFallback.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ReactElement } from 'react';
+import { useTimeout } from 'react-use';
+import styles from './App.module.css';
+
+interface Props {
+  isInspecting: boolean;
+  message?: string;
+}
+
+function LoadingFallback(props: Props): ReactElement {
+  const { isInspecting, message = 'Loading' } = props;
+
+  // Wait a bit before showing loader to avoid flash
+  const [isReady] = useTimeout(100);
```

Comment:
As for the value loader, I'm delaying showing the text `Loading...` by 100ms. I can't get rid of the flash until React Concurrent Mode is finally out, but I can make the flash less perceptible.